yt/yt/server/lib/exec_node/config.h

```diff
@@ -789,6 +789,13 @@ struct TExecNodeConfig
     //! Should include ChunkCache if artifacts are passed by symlinks.
     std::vector<NJobProxy::TBindConfigPtr> RootFSBinds;
 
+    //! Name of the user group which has access to container management. For instance, this group
+    //! should have access to containerd socket in case CRI is used. This value will only be used
+    //! if sidecars are to be spawned as part of the user job.
+    //! If unset, value `docker` will be used by default, leading to failure
```

Comment:
> leading to failure

That is not true, right? Let's elaborate that it will not fail here and may fail inside the job proxy.